RecoBTag/FeatureTools/src/deep_helpers.cc

```diff
@@ -119,4 +119,38 @@ namespace btagbtvdeep {
     return int16_t((qualityFlags & lostInnerHitsMask) >> lostInnerHitsShift) - 1;
   }
 
+  int center_norm_pad(const std::vector<float> &input,
+		      float center,
+		      float norm_factor,
+		      unsigned min_length,
+		      unsigned max_length,
+		      std::vector<float> *datavec,
```

Comment:
you could pass this as `std::vector<float>& datavec` to avoid dereferencing

Reply:
Yep, don't know why I did it the other way before